race_simulation.py

```diff
@@ -2,18 +2,23 @@
 import matplotlib.pyplot as plt
 import matplotlib.animation as animation
 
-# bounds of the room
-xlim = (0,30)
-ylim = (0,20)
+#~~~~~~Future Input Parameters~~~~~~
+distance = 400 # meters
+time     = 60  # seconds
```

Comment:
there are too many spaces here - this is not PEP8 compliant

Reply:
k